lib/SyTest/HomeserverFactory/Synapse.pm

```diff
@@ -109,7 +109,7 @@ sub get_options
 
    return (
       'dendron-binary=s' => \$self->{args}{dendron_binary},
-      'torture-replication+' => \$self->{args}{torture_replication},
+      'torture-replication:50' => \$self->{args}{torture_replication},
```

Comment:
The documentation you linked to seemed to suggest that the colon should be followed by a type, rather than a value?

Reply:
![image](https://user-images.githubusercontent.com/1389908/70903124-1d2b1500-1ff6-11ea-8c98-3209e61aa154.png)

Reply:
oooooooooooooooooooooooooooooooo

Reply:
that's special